dist/docker-images/ziti-controller/compose.yml

```diff
@@ -3,6 +3,10 @@ volumes:
   ziti-controller:
     driver: local
 
+networks:
+  ziti:
+    driver: bridge
```

Comment:
why was/is this necessary?

Reply:
Defining a custom network, "ziti" in this case, is a prerequisite to defining any compose service aliases, which is itself a prerequisite for ziti to have a single FQDN that is valid both inside and outside of the Docker network.

In other words, we're defining a DNS name for the controller that can be used by other containers inside the Docker network as well as all other clients outside the network.

This is the best way to enable containers inside the Docker network to operate in a ziti-normal way because it is non-trivial to enable hairpinning, and not possible to reliably enable hair-pinning at all.